src/app/view/components/container-selector/container-selector.component.scss

```diff
@@ -0,0 +1,54 @@
+@import "variables";
+@import "mixins";
+
+.container-selector {
+    height: 206px; 
+    border: 1px solid $gray-lighter; 
+}
+
+.head-container {
+    height: 48px; 
+    padding: $basic-padding $basic-padding * 2; 
+    background-color: $gray-lighter; 
+
+    .ui-button {
+        background: transparent;
```

Comment:
I don't think you need this if you use a flat button.

Reply:
Actually that's for searchable-dropdown button to match the mock.

Reply:
I run the project to see where is is button and I wansn't able to find it... can you point it to me?